src/index.ts

```diff
@@ -69,7 +69,6 @@ export interface RequestOptions {
     | 'BigtableTableAdminClient'
     | 'BigtableClient';
   reqOpts?: {};
-  retryOpts?: {};
```

Comment:
This _retryOpts_ property can be removed from the _RequestOptions_ interface. The _RequestOptions_ interface is only used for the request_ function and the request_ function doesn't make use of the _retryOpts_ property anymore.